nebula_tests/package.xml

```diff
@@ -12,6 +12,7 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
   <buildtool_depend>ros_environment</buildtool_depend>
 
+  <depend>autoware_perception_msgs</depend>
```

Comment:
This should probably be `autoware_sensing_msgs` now.
Since there aren't any code changes for `nebula_tests`, shouldn't this already be transitively included/linked through the dependencies on `nebula_ros`?

Reply:
Sorry, forgot to add the export, which was causing the need for those deps
Addressed in [a19eaf6](https://github.com/tier4/nebula/pull/284/commits/a19eaf603a82029b8aeeab1fd271744f8a9350d6)